npm.js

```diff
@@ -66,6 +73,8 @@ exports.translate = function(load){
 				if(pkg.browser){
 					delete pkg.browser.transform;
 				}
+				pkg = transform(loader, load, pkg);
```

Comment:
ok,
but i also dont know why we need the jsonTransformation in this situation (for fetching the plackages)? why is a custom tranform method needed in this situtation?

in https://github.com/stealjs/system-npm/pull/162/files#diff-e4bd8836ac7e24f5f5d08d5c984db99dR76 it is needed because we write that information out into the bundled file